README.md

```diff
@@ -58,7 +58,7 @@ The [react-native-tab-view](https://github.com/satya164/react-native-tab-view) e
 Open a Terminal in the project root and run:
 
 ```sh
-yarn add react-native-collapsible-segmented-view
+yarn add react-native-collapsible-segmented-view @react-native-community/segmented-control
 
 expo install @react-native-community/segmented-control
```

Comment:
hi @hamzawaleed0102 thanks for the PR, this package is already present in the next command (see line 63), with `expo install ...`

Reply:
Hi @PedroBern, since I was working on a non-expo/CLI react native project that's why I totally ignored the `expo install` command. And for RN CLI projects I think it's good idea to append the package in existing `yarn add` command.